lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -15,7 +15,7 @@ foam.CLASS({
     'org.chromium.webidl.URLExtractor',
   ],
 
-  imports: [ 'outputDAO?' ],
+  imports: [ 'outputDao? as dao' ],
```

Comment:
I believe sticking with import is a cleaner solution. If we drop the import, then each of the components will need to be explicitly passed the DAO on Pipeline creation. I feel like that would be a lot of redundant code and it would make the main script slightly uglier.